### PR TITLE
Dịch "batch-norm_vn" - Phần 1

### LINKED ISSUE #1373: Dịch "batch-norm_vn" - Phần 1
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_convolutional-modern/batch-norm_vn.md#L1-L50

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by DacDinh147**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #1373

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 1]
```diff
@@ -15,22 +15,23 @@ In this section, we describe batch normalization (BN) :cite:`Ioffe.Szegedy.2015`
...
-## *dịch tiêu đề phía trên*
+## Huấn luyện mạng học sâu
 
 <!--
 To motivate batch normalization, let us review a few practical challenges that arise
 when training ML models and neural nets in particular.
 -->
 
-*dịch đoạn phía trên*
+Để thấy được mục đích của việc chuẩn hóa theo batch, hãy cùng chúng tôi xem xét lại một vài vấn đề thực tế phát sinh khi huấn luyện mô hình ML (học máy) và mạng nơ-ron.
```
Comment (cuongvng) [inline chapter_convolutional-modern/batch-norm_vn.md:34]:
```suggestion
Để thấy được mục đích của việc chuẩn hóa theo batch, hãy cùng chúng tôi xem xét lại một vài vấn đề thực tế phát sinh khi huấn luyện các mô hình học máy và mạng nơ-ron.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 2]
```diff
@@ -45,7 +46,17 @@ Intuitively, we might conjecture that if one layer has activation values that ar
 This means that regularization becomes more critical.
 -->
 
-*dịch đoạn phía trên*
+1. Những lựa chọn liên quan đến việc tiền xử lý dữ liệu thường tạo nên sự khác biệt rất lớn trong kết quả cuối cùng.
+Hãy nhớ lại việc áp dụng mạng perceptron đa tầng để dự đoán giá nhà ở phần (:numref:`sec_kaggle_house`). 
+Việc đầu tiên khi chúng ta làm việc với dữ liệu thực tế là chuẩn hóa các đặc trưng đầu vào để chúng có giá trị trung bình bằng *không* và phương sai bằng *một*. 
+Một cách trực quan, việc chuẩn hóa này, hoạt động tốt với những bộ tối ưu bởi vì nó đặt giá trị các tham số làm một tiên nghiệm tại cùng một tỷ lệ.
```
Comment (cuongvng) [inline chapter_convolutional-modern/batch-norm_vn.md:52]: hm mình để ý thấy bạn hay dịch `intuition` và các dạng từ loại khác của từ này thành `trực quan`, không biết có hàm nghĩa gì khác ở đây không? Nghĩa của `intuition` mình hiểu là `trực giác`.
```suggestion
Theo trực giác, việc chuẩn hóa này hoạt động tốt với những bộ tối ưu vì nó đặt giá trị các tham số tiên nghiệm tại cùng một tỷ lệ.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 3]
```diff
@@ -45,7 +46,17 @@ Intuitively, we might conjecture that if one layer has activation values that ar
...
+2. Đối với mạng điển hình như MLP hay CNN, khi huấn luyện, hàm kích hoạt ở các tầng trung gian có thể nhận các giá trị với mức độ biến thiên lớn-
+dọc theo các tầng từ đầu vào cho đến đầu ra, qua các nút ở cùng một tầng, và theo thời gian do việc cập nhật giá trị tham số trong quá trình huấn luyện.
+Những nhà nghiên cứu phát minh ra kỹ thuật chuẩn hoá theo batch đã đưa ra một định đề chưa chính thức rằng sự thay đổi trong phân phối của những giá trị kích hoạt có thể làm cản trở sự hội tụ của mạng.
```
Comment (cuongvng) [inline chapter_convolutional-modern/batch-norm_vn.md:55]:
```suggestion
Những nhà phát minh ra kỹ thuật chuẩn hoá theo batch cho rằng sự thay đổi trong phân phối của những giá trị kích hoạt có thể cản trở sự hội tụ của mạng.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 4]
```diff
@@ -45,7 +46,17 @@ Intuitively, we might conjecture that if one layer has activation values that ar
...
+Một cách trực quan, chúng ta có thể phỏng đoán rằng nếu một tầng có những giá trị kích hoạt lớn gấp 100 lần so với các tầng khác, thì điều này bắt buộc cần phải có điều chỉnh bổ trợ trong tốc độ học.
```
Comment (cuongvng) [inline chapter_convolutional-modern/batch-norm_vn.md:56]:
```suggestion
Bằng trực giác, chúng ta có thể phỏng đoán rằng nếu một tầng có các giá trị kích hoạt lớn gấp 100 lần so với các tầng khác, thì có thể cần có các điều chỉnh bổ trợ trong tốc độ học.
```

File: chapter_convolutional-modern/batch-norm_vn.md (hunk 2 above)
Comment (DacDinh147) [reply chapter_convolutional-modern/batch-norm_vn.md:52]: @cuongvng mình nghĩ 2 từ cũng không có khác nhau lắm, mình dùng trực quan cho cảm tính, cho quan sát được mà không có chứng minh rõ ràng etc..

File: chapter_convolutional-modern/batch-norm_vn.md (hunk 4 above)
Comment (DacDinh147) [reply chapter_convolutional-modern/batch-norm_vn.md:56]: từ `necessitate` mang hàm ý bắt buộc

Comment (DacDinh147) [reply chapter_convolutional-modern/batch-norm_vn.md:56]: Mình sửa lại 1 ít theo bạn thế này nhé
```suggestion
Bằng trực giác,  chúng ta có thể phỏng đoán rằng nếu một tầng có các giá trị kích hoạt lớn gấp 100 lần so với các tầng khác, thì cần phải có các điều chỉnh bổ trợ trong tốc độ học.
```

File: chapter_convolutional-modern/batch-norm_vn.md (hunk 2 above)
Comment (DacDinh147) [reply chapter_convolutional-modern/batch-norm_vn.md]: @cuongvng  ở đây mình dùng `scale` là `thang lỷ lệ` thì có hợp lý hơn không.

File: chapter_convolutional-modern/batch-norm_vn.md (hunk 4 above)
Comment (cuongvng) [reply chapter_convolutional-modern/batch-norm_vn.md:56]: ok mình đồng ý.

File: chapter_convolutional-modern/batch-norm_vn.md (hunk 2 above)
Comment (cuongvng) [reply chapter_convolutional-modern/batch-norm_vn.md]: mình nghĩ `tỉ lệ` và `thang tỉ lệ` trong ngữ cảnh này cũng không khác nhau nhiều.

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 5]
```diff
@@ -15,22 +15,23 @@ In this section, we describe batch normalization (BN) :cite:`Ioffe.Szegedy.2015`
 Together with residual blocks—covered in :numref:`sec_resnet`—BN has made it possible for practitioners to routinely train networks with over 100 layers.
 -->
 
-*dịch đoạn phía trên*
-
-
+Việc huấn luyện mạng nơ-ron học sâu thì không dễ. 
+Và để cho mô hình hội tụ trong một khoảng thời gian hợp lý là một vấn đề hóc búa.
+Trong phần này, chúng ta sẽ đề cập đến việc chuẩn hóa theo batch (BN) :cite:`Ioffe.Szegedy.2015`, một kỹ thuật phổ biến và hiệu quả nhằm giúp tăng tốc độ hội tụ của mạng học sâu một cách ổn định.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:20]:
```suggestion
Trong phần này, chúng ta sẽ đề cập đến việc chuẩn hóa theo batch (_Batch Normalization - BN_) :cite:`Ioffe.Szegedy.2015`, một kỹ thuật phổ biến và hiệu quả nhằm giúp tăng tốc độ hội tụ của mạng học sâu một cách ổn định.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 6]
```diff
@@ -15,22 +15,23 @@ In this section, we describe batch normalization (BN) :cite:`Ioffe.Szegedy.2015`
...
+Cùng với các khối thặng dư được đề cập ở phần :numref:`sec_resnet`—BN đã giúp những người thực nghiệm có thể huấn luyện mạng học sâu với hơn 100 tầng.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:21]:
```suggestion
Cùng với các khối thặng dư được đề cập ở phần :numref:`sec_resnet` — BN đã giúp những người hành nghề có thể huấn luyện mạng học sâu với hơn 100 tầng một cách đều đặn.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 7]
```diff
@@ -15,22 +15,23 @@ In this section, we describe batch normalization (BN) :cite:`Ioffe.Szegedy.2015`
...
-## *dịch tiêu đề phía trên*
+## Huấn luyện mạng học sâu
 
 <!--
 To motivate batch normalization, let us review a few practical challenges that arise
 when training ML models and neural nets in particular.
 -->
 
-*dịch đoạn phía trên*
+Để thấy được mục đích của việc chuẩn hóa theo batch, hãy cùng chúng tôi xem xét lại một vài vấn đề thực tế phát sinh khi huấn luyện các mô hình học máy và mạng nơ-ron.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:34]:
```suggestion
Để thấy được mục đích của việc chuẩn hóa theo batch, hãy cùng chúng tôi xem xét lại một vài vấn đề thực tế phát sinh khi huấn luyện các mô hình học máy và đặc biệt là mạng nơ-ron.
```
